<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageEdges</class>
 <widget class="QDialog" name="ManageEdges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="edges_table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="from_stadium_cbox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="to_stadium_cbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="from_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="cost">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cost_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="row_to_remove">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="row_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Row</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
